source/reference/disaster-recovery.txt

```diff
@@ -15,9 +15,10 @@ Use ``mongosync`` for Disaster Recovery
 Setting up {+c2c-product-name+} to prepare for disaster recovery follows the 
 same procedure as other :ref:`connections <c2c-connecting>` between clusters.
 Synchronization keeps the destination cluster up to date with the source
-cluster and ready as a backup. In the event that the source cluster fails or
-becomes otherwise unavailable, the destination cluster can replace it as the
-primary cluster.
+cluster using :term:`eventual consistency`. If the source cluster 
```

Comment:
Tim made a suggestion on the ticket to add some extra details about eventual consistency here. We could insert a sentence like "This means that until the migration is committed, the destination cluster may represent a stale and inconsistent version of the source cluster", but I'll leave it up to your discretion since the following sentence states this as well.

Reply:
Thanks for bringing this up! Adding this in.